nix/cardano-services/deployments/default.nix

```diff
@@ -242,104 +242,47 @@ in
           };
         };
 
-        "dev-sanchonet@us-east-1@v1" = final: {
+
+        "dev-sanchonet@us-east-1" = final: {
+          name = "${final.namespace}-cardanojs";
           namespace = "dev-sanchonet";
-          name = "${final.namespace}-cardanojs-v1";
           network = "sanchonet";
           region = "us-east-1";
 
           providers = {
             backend = {
               enabled = true;
-              env.USE_SUBMIT_API = "true";
-              env.USE_BLOCKFROST = lib.mkForce "false";
-              env.SUBMIT_API_URL = "http://${final.namespace}-cardano-stack.${final.namespace}.svc.cluster.local:8090";
```

Comment:
dont we need submitapi and blockfrost?

Reply:
I will clarify with @mirceahasegan and will add that in a new PR, I would like to merge this one to unblock dev-preview deploys